library/src/main/java/org/xmtp/android/library/Group.kt

```diff
@@ -215,6 +220,38 @@ class Group(val client: Client, private val libXMTPGroup: FfiGroup) {
         return libXMTPGroup.updateGroupName(name)
     }
 
+    fun isAdmin(inboxId: String): Boolean {
+        return libXMTPGroup.isAdmin(inboxId)
+    }
+
+    fun isSuperAdmin(inboxId: String): Boolean {
+        return libXMTPGroup.isSuperAdmin(inboxId)
+    }
+
+    suspend fun addAdmin(inboxId: String) {
+        return libXMTPGroup.addAdmin(inboxId)
```

Comment:
Do these error if they don't have permission to add an admin? Wondering if we should catch and surface a more friendly XMTPException?

Reply:
Added error handling here and made an issue to remind myself to improve the error messaging in libxmtp so we can surface more specific errors - 115b370bf624440cf8e4664597b5524a2f033471